packages/blade/src/components/Drawer/_decisions/decisions.md

```diff
@@ -0,0 +1,174 @@
+# Drawer
+
+A drawer is a panel that slides in mostly from the right side of the screen over the existing content in the viewport. It helps in providing additional details or context and can also be used to promote product features or new products.
+
+<img width="500px" src="./2024-02-07-20-04-07.png" alt="Drawer Figma Skeleton">
+
+## Design
+
+- [Drawer - Figma Design](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade-DSL?node-id=78667%3A66663&mode=dev)
+
+## Proposed API
+
+```jsx
+<Drawer
+  isOpen={}
+  onDismiss={() => {}}
+  showOverlay
+>
+  <DrawerHeader
```

Comment:
Will this be fixed while scrolling?

Reply:
Good question. Header scrolls right now. Let me confirm with designer once.

Reply:
Confirmed. It should be fixed on top similar to how we have on Modal